### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Sumit-pd**
Updated contribution.md 

## Description
The details of the values `<USE_EMULATOR>` and  `<CYPRESS_PROJECT_ID>` was not mentioned in the contribution.md so I added the same

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x]  Documentation update

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,14 +42,16 @@ To install packages, run
...
+6. Paste the configuration `.env` file. **(this will be found in the project settings section of firebase cloud)**
 7. You can find your `<FIREBASE_DATABASE_URL>` in the Realtime Database section of the Firebase console. Depending on the location of the database, the database URL  will be in one of the following forms:
     - `https://DATABASE_NAME.firebaseio.com` **(for databases in us-central1)**
     - `https://DATABASE_NAME.REGION.firebasedatabase.app`**(for databases in all other locations)**
 8. You can get your `<FIREBASE_VAPID_KEY>` from Cloud-Messaging tab
     - navigate to the setting of your project Open the Cloud Messaging tab.
     - scroll to the Web configuration section.
     - In the Web Push certificates tab, click Generate Key Pair. The console displays a notice that the key pair was generated. You get your Vapid key form there.
+9. set `<USE_EMULATOR>` to true 
+10. You can your `<CYPRESS_PROJECT_ID>` as cypress project id from cypress cloud
```
Comment (Sarfraz-droid) [inline CONTRIBUTING.md:54]: link it to cloud.cypress.io

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,14 +42,16 @@ To install packages, run
...
-6. Paste the configuration `.env` file.
+6. Paste the configuration `.env` file. **(this will be found in the project settings section of firebase cloud)**
 7. You can find your `<FIREBASE_DATABASE_URL>` in the Realtime Database section of the Firebase console. Depending on the location of the database, the database URL  will be in one of the following forms:
     - `https://DATABASE_NAME.firebaseio.com` **(for databases in us-central1)**
     - `https://DATABASE_NAME.REGION.firebasedatabase.app`**(for databases in all other locations)**
 8. You can get your `<FIREBASE_VAPID_KEY>` from Cloud-Messaging tab
     - navigate to the setting of your project Open the Cloud Messaging tab.
     - scroll to the Web configuration section.
     - In the Web Push certificates tab, click Generate Key Pair. The console displays a notice that the key pair was generated. You get your Vapid key form there.
+9. set `<USE_EMULATOR>` to true
```
Comment (Sarfraz-droid) [inline CONTRIBUTING.md:53]: Add. 
As you're using emulator, set <USE_EMULATOR> to true

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Sumit-pd) [reply CONTRIBUTING.md:54]: ok working on it

**Comment by Sumit-pd** [comment]
@Sarfraz-droid can review the changes!

**Review by Sarfraz-droid — APPROVED** [review]

**Pull request merged**